server/src/utils/Filter.js

```diff
@@ -125,7 +125,7 @@ class Filters {
             query.where(builder => {
                 builder.where(nameToUse, toSQLOperator(operator, value), value)
                 if (settings.includeEmpty) {
-                    builder.orWhereRaw(`nullif( ??, ' ') is null`, nameToUse)
+                    builder.orWhereRaw(`nullif( ??, '') is null`, nameToUse)
```

Comment:
Do we want to catch all empty strings?  If so we could use [`TRIM`](https://www.w3schools.com/sqL/func_sqlserver_trim.asp) (only trims spaces though)

```suggestion
                    builder.orWhereRaw(`nullif( trim( ?? ), '') is null`, nameToUse)
```

Reply:
I originally included that, but figured we rather want to be more strict on the allowed format and do this in the application-layer with Joi-validation.

Reply:
OK sure!